terraform/modules/data-share-service/cloudwatch.tf

```diff
@@ -41,23 +41,43 @@ module "metrics_dashboard" {
       stat   = "Sum",
       metrics = [
         {
-          name       = "API_CALLS.PublishEvent.count",
-          dimensions = { success = true },
+          name = "count.eventsController.publishEvent.count",
+          dimensions = {
+            result    = "success",
+            exception = "none",
+            method    = "publishEvent",
+            class     = "uk.gov.gdx.datashare.controllers.EventsController"
+          },
           attributes = { label = "PublishEvent" }
         },
         {
-          name       = "API_CALLS.GetEvent.count",
-          dimensions = { success = true },
+          name = "count.eventsController.getEvent.count",
+          dimensions = {
+            result    = "success",
+            exception = "none",
+            method    = "getEvent",
+            class     = "uk.gov.gdx.datashare.controllers.EventsController"
+          },
           attributes = { label = "GetEvent" }
         },
         {
-          name       = "API_CALLS.GetEvents.count",
-          dimensions = { success = true },
+          name = "count.eventsController.getEvents.count",
```

Comment:
Is this different from http.server.requests.count where exception = 'none' and method = 'GET' and uri = '/events' and outcome = 'SUCCESS'? Or are we counting the number of individual events somehow?

Reply:
Nope there is no difference here, my thought is that we want a consistent way of generating our metrics, and that we don't want to grab some from the automagic ones, and some where we define them

Reply:
This is still automagic, just in a worse way IMO

Reply:
Sure